Src/Utils/instancelessmethod.py

```
@@ -0,0 +1,19 @@
+from functools import wraps
+
+
+class instancelessmethod:
+    """
+    Декоратор для вызова методов класса, который передаёт в self значения в зависимости от типа вызова.
+    Он делает метод способным к вызову как из класса, так и из объекта.
+    Если вызывают из класса, то self является самим классом, а если из объекта, то self является объектом.
```

Ruff (0.15.17)

[warning] Docstring contains ambiguous `а` (CYRILLIC SMALL LETTER A). Did you mean `a` (LATIN SMALL LETTER A)?

(RUF002)